modules/web-console/frontend/app/components/page-configure-advanced/style.scss

```diff
@@ -94,40 +94,3 @@ page-configure-advanced {
         margin-top: -10px;
     }
 }
-
```

Comment:
Is this edition relevant to font changing? Or is it just a side cleanup?

Reply:
I found that the .pca-panel class was left unused, so instead of removing "font-family", I removed the whole thing.

Reply:
Ok